README.md

```diff
@@ -82,13 +85,17 @@ After satisfying the above requirements, execute the following command from the
 By default, this will set up an instance of rawls pointing to the database and Sam in dev. 
 It will also set up a process that will watch the local files for changes, and restart the service when source files change.
 
-See docker-rsync-local-rawls.sh for more configuration options.
+See `docker-rsync-local-rawls.sh` for more configuration options.
+
+When Rawls starts up, access the Rawls Swagger page: https://local.dsde-dev.broadinstitute.org:20443/
 
-If Rawls successfully starts up, you can now access the Rawls Swagger page: https://local.dsde-dev.broadinstitute.org:20443/
+### Back Rawls
 
-### Useful Tricks:
+#### Additional setup for Back Rawls
 
-#### Front & Back Rawls
+1. Personal clone of the [Rawls Dev database](https://console.cloud.google.com/sql/instances/terraform-qfarbdq3lrexxck5htofjs5z6m/overview?project=broad-dsde-dev)
```

Comment:
why are we recommending a personal clone?

Reply:
You can interfere with Dev when testing if not using a clone. We had this issue with Leo and more recently with Rawls, it is safer to create a clone and then stop it after testing to avoid cost

Reply:
>1 back Rawls sharing a database act as consumers that compete over grabbing & submitting workflows.

This caused a ton of confusion where we believed local back Rawls was working; but it was actually Dev back Rawls stealing its workflows.